packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -1445,6 +1494,22 @@ class DescopeWc extends BaseDescopeWc {
       });
 
     this.#addPasscodeAutoSubmitListeners(next);
+
+    if (this.isDismissScreenErrorOnInput) {
+      // listen to all input events in order to clear the global error state
+      this.contentRootElement.querySelectorAll('[name]').forEach((ele) => {
```

Comment:
![image](https://github.com/user-attachments/assets/cd0c0ee8-794c-42c1-ac88-50f6db18470d)

shouldn't we use this selector?

Reply:
I looked and its only the phone inputs which we should consider in this case,

Reply:
It's the phone on `page-editor-components` which is the V1 components

Reply:
I updated the selector to be on the safe side